.github/workflows/matlab.yml

```
@@ -0,0 +1,20 @@
+name: matlab graphic generation
+on:
+  push:
+    branches:
+      -main
+      -pjmbs_matlab_actions
```

Leave only the main branch and add the pull request trigger

```suggestion
on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]
```